src/makeCalculator.js

```diff
@@ -4,7 +4,29 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: (a, b) => a + b,
```

Comment:
The `add` method should update the `result` property of the calculator object. Consider using `this.result += a + b;` to accumulate the result.

Reply:
how to do this? like this:
add: (a, b) => this.result += a + b;